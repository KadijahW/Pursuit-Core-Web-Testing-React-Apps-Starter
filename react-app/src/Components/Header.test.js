import React from 'react';
import { render } from '@testing-library/react';
import Header from './Header';

describe('Header component', () => {
  test('Renders Go Fund Me header', () => {
    const { container, getByRole } = render(<Header />)
    const header = getByRole('heading')
    expect(header.textContent).toBe('Go Fund Mo')
    expect(container).toMatchSnapshot()
  })
})

