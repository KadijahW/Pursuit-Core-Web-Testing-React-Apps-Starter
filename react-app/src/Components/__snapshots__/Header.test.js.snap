// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Renders Go Fund Me header 1`] = `
<div>
  <section
    class="jumbotron text-center"
  >
    <h1
      class="jumbotron-heading"
    >
      Go Fund Mo
    </h1>
    <p
      class="lead text-muted"
    >
      Help me go on a vacation to a beach somewhere
    </p>
  </section>
</div>
`;
